pkg/debugapi/topology.go

```diff
@@ -12,7 +12,7 @@ import (
 )
 
 type topologyResponse struct {
-	Topology string `json:"topology"`
+	Topology string `json:"topology"` //TODO: this is not so great since it makes the actual kademlia state to be an escaped string
```

Comment:
@janos WDYT about this?

Reply:
see the change in https://github.com/ethersphere/bee/pull/188/commits/8f9c45951b96194f84a697ec39cf5e4624db37e0

Reply:
Yes, I actually did have a comment on this in my not submitted pr review to do something similar as in your changeset. I also think that it would work with just `jsonhttp.OK(w, ms)` and explicit call to MarshalJSON.